_ja/functions/findLast.md

```diff
@@ -0,0 +1,15 @@
+---
+name: findLast
+---
+
+# `findLast`
+
+@include [signatures/findLast.md]
+
+`find` は、述語 `p` を満たす最後の要素を `Some` に包んで返します。
```

Comment:
Wrong function name.

Reply:
Fixed in https://github.com/superruzafa/visual-scala-reference/pull/11/commits/5d802388170b18a0bf3b64bcacb31fbcfb97c371